docs/contributing/backend.md

```diff
@@ -0,0 +1,183 @@
+# Backend
+
+This guide is designed to support both Coder engineers and community contributors in understanding our backend systems and getting started with development.
+
+Coder’s backend powers the core infrastructure behind workspace provisioning, access control, and the overall developer experience. As the backbone of our platform, it plays a critical role in enabling reliable and scalable remote development environments.
+
+The purpose of this guide is to help you:
+
+* Understand how the various backend components fit together.
+* Navigate the codebase with confidence and adhere to established best practices.
+* Contribute meaningful changes - whether you're fixing bugs, implementing features, or reviewing code.
+
+By aligning on tools, workflows, and conventions, we reduce cognitive overhead, improve collaboration across teams, and accelerate our ability to deliver high-quality software.
+
+Need help or have questions? Join the conversation on our [Discord server](https://discord.com/invite/coder) — we’re always happy to support contributors.
+
+## Quickstart
+
+To get up and running with Coder's backend, make sure you have **Docker installed and running** on your system. Once that's in place, follow these steps:
+
+1. Clone the Coder repository and navigate into the project directory:
+
+```sh
+git clone https://github.com/coder/coder.git
+cd coder
+```
+
+1. Run the development script to spin up the local environment:
```

Comment:
Perhaps we could add a quick explainer what the develop script does to deepen the understanding? (I.e. spins up coderd with embedded postgresql, creates an admin and a member user, prepares an initial template, etc.)

Also, are all of these intentionally `1.`?